src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -590,10 +685,9 @@ internal async void ExecutePackageDownload(string name, PackageVersion package,
                     }
                 }).ToList();
 
-                // if any header download fails, abort
-                if (dependencyVersionHeaders.Any(x => x == null))
-                {
-                    return;
+                if (!dependencyVersionHeaders.Any(x => x.name == name))
+                {// Add the main package if it does not exist
+                    dependencyVersionHeaders.Add(package);
```

Comment:
some packages (on dynamopackages.com) do not have the main package in the full_dependency_ids